lib/Logger.tests.js

```diff
@@ -0,0 +1,97 @@
+import expect, { spyOn, restoreSpies } from 'expect';
+import Logger from './Logger';
+describe('Logger', () => {
+  afterEach(() => {
+    restoreSpies();
+  });
+
+  describe('server', () => {
+    it('should instantiate with loggly', () => {
+      const logger = new Logger({
+        loggly: {
+          subdomain: 'foo',
+          token: 'bar',
+          tags: 'baz,bing',
+        },
+      });
+
+      expect(Object.keys(logger.winston.transports)).toEqual(['console', 'loggly']);
+    });
+
+    it('should not instantiate with loggly if missing config', () => {
+      const logger = new Logger();
+      expect(Object.keys(logger.winston.transports)).toEqual(['console']);
+    });
+
+    it('should make call to winston with correct args', () => {
+      const logger = new Logger({
+        meta: {
+          boop: 'scoop',
+        },
+        loggly: {
+          subdomain: 'foo',
+          token: 'bar',
+          tags: 'baz,bing',
+        },
+      });
+      spyOn(logger.winston.transports.console, 'log').andReturn(true);
+      spyOn(logger.winston.transports.loggly, 'log').andReturn(true);
+      logger.log('info', 'foo');
+      expect(logger.winston.transports.console.log).toHaveBeenCalled();
+      const args = logger.winston.transports.console.log.calls[0].arguments;
+      expect(args[0]).toEqual('info');
```

Comment:
It might be overkill but having tests iterating over the different methods we care about ['info', 'debug', 'fatal'....] could be useful to guarantee that these methods all work.